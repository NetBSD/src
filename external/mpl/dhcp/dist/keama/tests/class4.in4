# class declaration config

# options
option mysystem code 250 = text;
option myversion code 251 = unsigned integer 16;

# class declaration
class "foobar" {
    match if option mysystem = "version1";
    option myversion 1;
}
