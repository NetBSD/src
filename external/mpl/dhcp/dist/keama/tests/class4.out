{
  # class declaration config
  # options
  "Dhcp4": {
    "option-def": [
      {
        "space": "dhcp4",
        "name": "mysystem",
        "code": 250,
        "type": "string"
      },
      {
        "space": "dhcp4",
        "name": "myversion",
        "code": 251,
        "type": "uint16"
      }
    ],
    "client-classes": [
      # class declaration
      {
        "name": "foobar",
        /// from: match if (option dhcp.mysystem) = 'version1'
        "test": "option[250].hex == 'version1'",
        "option-data": [
          {
            "space": "dhcp4",
            "name": "myversion",
            "code": 251,
            "data": "1"
          }
        ]
      }
    ]
  }
}
